src/db/user_in_group.js

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
This file was (rightfully) removed in 618b4c8b2996c8b54668e98e06591150ec931d17. Its functionality should be moved to db/group and/or db/user